cds-snc/cppd-medical-report-sails
# Applicants understand what to do with the code

# What this does
Updates the Invite screen according to design [changes described here](https://trello.com/c/JGD3YPhx/500-applicants-understand-what-to-do-with-the-code).

## To test
- Start a new application
- Complete the Personal and Consent forms
- See the new Invite screen
